test/browser/pages/error.js

```diff
@@ -7,11 +7,11 @@ module.exports = class PlaywrightDevPage {
   }
 
   getSomethingWentWrongMessage() {
-    return "Sorry, there is a problem with the service";
+    return "Sorry, there is a problem – Prove your identity – GOV.UK";
   }
 
   getErrorTitle() {
-    return this.page.textContent('[data-id="error-title"]');
```

Comment:
This title is just "Sorry, there is a problem" in the msgs file, should the "Prove your identity – GOV.UK" be at the end aswell so match the getSomethingWentWrongMessage above?